modules/ROOT/pages/ch2-api.adoc

```diff
@@ -0,0 +1,376 @@
+** Configure external log forwarding.
+** Deletes the external log forwarding configuration.
+
+* *TLS Contexts*
+** Get the list of TLS contexts inside the private space.
+** Create a TLS context.
+** Delete the TLS context.
+** Get the TLS context.
+** Update an existing TLS context.
+
+* *Runtime Fabrics*
+** Get the list of Runtime Fabrics in the organization.
+** Runtime Fabric register endpoint.
+** Delete associated apps.
+** Get capacity information about the runtime fabric for an environment.
+** Update existing fabric.
+** Get a detailed health check for the Runtime Fabric.
+** Get the list of Runtime Fabric Groups.
+** Get the list of environment associations.
+** Create a new environment association.
+** Update an existing environment association.
+** Retrieve short-lived credential to allow pulling Runtime Fabric Docker images.
+** Retrieve properties related to image registry.
+
+* *Upgrade Compatibility* 
+** Get pre-upgrade compatibility check status and optionally limit the scope of the pre-check operation.
+** Run the pre-upgrade compatibility checks.
+
+* *Mule Runtime*
+** Retrieve all supported Mule versions.
++
+The first version returned is the most recent, recommended version, and the last version returned is the oldest supported version
+** Retrieve a single name. *what?*
+** Retrieves tags for a name. *what?*
+** Retrieve a single name for a name. *what?*
+
+* *Downloads*
+** List the available downloads.
+** Download the latest installer scripts.
+** Download the latest installer package
+** Get the pre-signed URL of the latest installer package.
+** Download the latest Runtime Fabric agent helm chart.
+** Download the latest rtfctl binary.
+
+////
+
+
+== CloudHub 1.0 
+
+*THIS IS THEORETICAL, BASED ON CH 1.0* 
+
+
+The CloudHub 2.0 management API (CloudHub 2.0 API) enables you to programmatically access the functions of Anypoint Runtime Manager, such as:
+
+* Manage applications:
+** Create, deploy, start, stop, list, and delete an application on CloudHub 2.0.
+** Update application metadata, including the number of replicas, Mule runtime engine version, and system properties.
+** Retrieve statistics for an application or replica.
+** Retrieve application transactions and events for a transaction.
+** Check for replay data, replay the flow for a transaction, and delete replay data for an application.
+** Manage static IP addresses and release unused IP addresses associated with an application.
+** Download the Mule package file for the application.
+** Update, enable, disable, or run all or some schedules.
+
+
+* Manage logs, notifications, and alerts:
+** Add, update, or retrieve log levels for an application.
+** Download the log file for an application or instance.
+** Create, update, or retrieve a notification.
+** Mark a notification as read or unread.
+** Create, update, or delete an alert triggered by an application.
+** Retrieve an alert and alert history.
```

Comment:
These will be performed through Application Manager API. For more details, please consult the Application Manager team.